node/tests/SharedTests.ts

```diff
@@ -1046,6 +1046,72 @@ export function runBaseTests<Context>(config: {
         config.timeout,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
```

Comment:
Add a cross slot test for this command https://github.com/aws/glide-for-redis/blob/64d2273f093a6f27d0674f741f6ea7303b02563b/node/tests/RedisClusterClient.test.ts#L279